config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,20 @@
+FriendlyId.defaults do |config|
+  config.use :reserved
```

Comment:
This is the default config from the generator, just with the boilerplate text removed, and some reformatting of the lines.